internal/pkg/log/logformat.go

```diff
@@ -0,0 +1,29 @@
+package log
+
+import (
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
+)
+
+type LogFormat int
+
+const (
+	Console LogFormat = iota
+	JSON
```

Comment:
Should I prefix the names to be LogFormatConsole and LogFormatJSON instead? Console and JSON feel like they might easily conflict with something in the future.